docs/concepts/Dialog-Command.md

```diff
@@ -0,0 +1,78 @@
+# Dialog Command
+
+`askx dialog` opens a interactive mode, similar to Node's REPL, in which developers can simulate a multi-turn conversation with  Alexa. For each utterance, Dialog command leverages [simulate-skill](https://developer.amazon.com/en-US/docs/alexa/smapi/ask-cli-command-reference.html#simulate-command) to asynchronously poll simulation result until it's available and shows Alexa text responses. **Note:** Skill must have been enabled for simulation before running the dialog command.
```

Comment:
* typo: opens `an` interactive mode
* expand REPL by showing both its full name and acronym
* please put the last sentence as a separate bullet point, as the pre-requisite lists to use the dialog command, such as skill type and skill locales